### PR TITLE
Left side internship page

### PR DESCRIPTION
**Pull request by GhBogdan97**
internship details + cv download

### CODE REVIEW

File: app/components/elements/Testimonials/index.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
+.container {
+  margin: 0 auto;
+  padding: 40px;
+  width: 80%;
+  color: #333;
+  background: #419be0;
```
Comment (nicucomiati990) [inline app/components/elements/Testimonials/index.css:6]: please define these colors in the constants file and use them from there

File: app/containers/elements/StudentManagement/index.js [hunk 2]
```diff
@@ -55,119 +64,140 @@ export class StudentManagement extends React.Component
...
+
+  showCv(cv) {
+    const file = new Blob([cv.Cv], { type: 'application/pdf' });
+  }
+
+  render() {
+    const applications = Object.values(this.props.applications.applications);
+    console.log(this.props);
+    const availability = this.props.applications.availability;
+    if (
```
Comment (nicucomiati990) [inline app/containers/elements/StudentManagement/index.js]: you can simplify this as if (!(applications && applications.length))

**Review by LauraS97 — APPROVED** [review]

**Review by Alex0719 — APPROVED** [review]
Nice work!

File: app/components/elements/Testimonials/index.css (hunk 1 above)
Comment (GhBogdan97) [reply app/components/elements/Testimonials/index.css:6]: erau temporar, nu o sa ramana asa :D

**Review by LauraS97 — APPROVED** [review]

**Pull request merged**